backend/django/core/templates/projects/detail.html

```diff
@@ -263,10 +266,16 @@ <h5 class="panel-title">
             </a>
             {% endif %}
           {% if project.has_model %}
-            <a class="btn btn-primary" id="download_model_btn">Download Model and Labeled Data</a>
+            <a class="btn btn-primary" id="download_model_btn" onclick="download_model({{project.pk}},1)">Download Model and All Labeled Data</a>
+            {% if project.unverified_labeled_data_count > 0 %}
+              <a class="btn btn-primary" id="download_model_btn_verified" onclick="download_model({{project.pk}},0)">Download Model and Only Verified Labeled Data</a>
+            {% endif %}
           {% else %}
             {% if project.labeled_data_count > 0 %}
-            <a class="btn btn-primary" id="download_btn">Download Labeled Data</a>
+            <a class="btn btn-primary" id="download_btn" onclick="download_data({{project.pk}},1)">Download Labeled Data</a>
```

Comment:
The buttons on the project detail page; however, are correctly downloading data.